### ISSUE TITLE
Bug in options from strings

### ISSUE THREAD

**Issue by alpha-and-omega** [lindell/JsBarcode#228]
I don't know if this documented, but "width" can be float (this is required to draw fitted svg barcodes). But since it works using options it should work using jsbarcode-width attribute, but it don't.
This is due to bug in src/help/optionsFromStrings.js
parseInt should be replaced with parseFloat for "jsbarcode-width" option.

**Comment by SanichKotikov** [comment]
Hi. I'm not sure that float values is a good idea for width param.

**Comment by alpha-and-omega** [comment]
If you don't like it, just don't touch it! I have nothing against it, for me this is good idea.
Nobody uses barcodes for images, barcodes are used for printing.

**Comment by Soheevich** [comment]
Yep, it would be great to use float values. I need to print barcodes on pricetags with precise dimensions. But I can't use float values in strings. It turns that I need to invoke init() for every barcode wich I don't want to do.
